rust/main/hyperlane-base/src/cache/moka/metered_cache.rs

```diff
@@ -0,0 +1,104 @@
+use async_trait::async_trait;
+use derive_builder::Builder;
+use derive_new::new;
+use hyperlane_core::H256;
+use maplit::hashmap;
+use prometheus::IntCounterVec;
+use serde::{de::DeserializeOwned, Serialize};
+use std::fmt::Debug;
+
+use crate::cache::HyperlaneCache;
+
+use super::{CacheResult, Expiration};
+
+/// Basic cache information.
+#[derive(Debug, Clone)]
+pub struct MeteredCacheConfig {
+    /// The name of the cache set on creation.
+    pub cache_name: &'static str,
```

Comment:
I wonder why we need to make it `'static`?

Reply:
As discussed, changed `cache_name` to String to avoid needing to to be `'static`

[Fix](https://github.com/Mantas-M/hyperlane-monorepo/commit/5ddbe070c107f7c51f27f645cc6dbe7b3db389a8)